tests/wpt/metadata/XMLHttpRequest/formdata.htm.ini

```diff
@@ -1,10 +1,5 @@
 [formdata.htm]
   type: testharness
-  [formdata with string]
-    expected: FAIL
-
-  [formdata with named string]
-    expected: FAIL
 
   [formdata from form]
     expected: FAIL
```

Comment:
Because I didn't support it in this PR yet ... but looks not difficult ... I will see what I can do